### PR TITLE
VaultBet project submission

### PR DESCRIPTION
**Pull request by welchysquelchy**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

**Comment by sa-github-api** [comment]
Dear @welchysquelchy,

In order to potentially merge your code in this open-source repository and therefore proceed with your contribution, we need to have your approval on DFINITY's [CLA](https://github.com/dfinity/cla/blob/main/CLA.md).

If you decide to agree with it, please visit this [issue](https://github.com/dfinity/cla/issues/455) and read the instructions there. Once you have signed it, re-trigger the workflow on this PR to see if your code can be merged.

— The DFINITY Foundation

File: showcase.json [hunk 1]
```diff
@@ -3690,5 +3690,27 @@
...
+  },
+  {
+    "id": "vaultbet",
+    "name": "VaultBet",
+    "oneLiner": "Redefining online betting on the Internet Computer.",
+    "website": "https://vault-bet.com",
+    "tags": [
+      "Games",
+      "DeFi",
+      "Bitcoin",
```
Comment (jennifertrin) [inline showcase.json:3702]: Can you please confirm how you include Bitcoin, Ethereum, and NFTs? I was unable to find features related to these on the platform.

File: showcase.json [hunk 2]
```diff
@@ -3690,5 +3690,27 @@
...
+      "NFT"
+    ],
+    "twitter": "https://twitter.com/VaultBet",
+    "discord": "https://discord.gg/qmr92dup3J",
+    "description": "VaultBet is redefining online betting on the Internet Computer. Hosted completely on-chain and built on the IC. Whether it's sports, lottery or arcade - users can experience transparency, security, and community-driven gambling like never before.",
+    "usesInternetIdentity": false,
+    "stats": "1000+ Monthly Users",
+    "display": "Normal",
+    "logo": "/img/showcase/vaultbet_logo.webp",
+    "screenshots": [],
```
Comment (jennifertrin) [inline showcase.json:3713]: Can you please remove any key pairs that are blank?

File: showcase.json (hunk 1 above)
Comment (welchysquelchy) [reply showcase.json:3702]: Hello, yes, users of our website are able to play our arcade games with ICP, ckBTC and ckETH as well. We also did a crowdfunding campaign where our supporters were awarded NFTs that granted them a number of rewards such as the ability to claim monthly VBT (our own token) which can be used to play on our site as well. If you go to the arcade section on our site you will have the option to deposit and use ICP, ckBTC, ckETH, and VBT for playing. To claim VBT you must connect to our site with a wallet holding at least one of our Market Maker or Pro Player NFTs.

File: showcase.json (hunk 2 above)
Comment (welchysquelchy) [reply showcase.json:3713]: Yes sure

**Review by jennifertrin — APPROVED** [review]

**Pull request merged**